### PR TITLE
adding license

### PR DESCRIPTION
**Pull request by SaraOgaz**
@justincely

### CODE REVIEW

**Comment by nden** [comment]
Hey @SaraOgaz and @justincely , Can you clarify this new copyright notice. I've only seen an AURA one. When are we supposed to use this vs the AURA one?

**Comment by justincely** [comment]
@nden, talked to @arfon about this earlier today and this was the DSMO's initial recommendation.  I'm following up with BRC to verify that they're OK with this plan.  Should have a final answer for you and the branch tomorrow - hoping we can just finish this once and for all.

**Comment by jhunkeler** [comment]
@justincely When rendered in github's rst engine the license file looks pretty messed up (i.e. no bullet points, and has compacted lines where carriage returns used to be).

**Comment by arfon** [comment]
I would recommend just keeping this as a plain text file called `LICENSE`

**Comment by justincely** [comment]
see #59 for fixed format.  @arfon - ours have typically been .rst or .md - what's the benefit of plaintext?

**Comment by arfon** [comment]
> what's the benefit of plaintext?

None, just personal preference :-) Also, it's not really a `.md` or `.rst` file, the licenses are plain text.

On GitHub it also has the benefit of not triggering something like the Markdown or RST pipeline from trying to interpret syntax in the files. Thinking about it, by my own logic `LICENSE` or `LICENSE.txt` would probably be best.

**Comment by SaraOgaz** [comment]
I'm with @arfon on this one, I can put this change in on my next PR if that's good with you @justincely

File: LICENSE.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Copyright (C) 2016 Space Telescope Science Institute (STScI)
+
+Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+
+Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
```
Comment (hbushouse) [inline LICENSE.rst]: Can I get support for an Oxford comma preceding the last "and" clause in lines 5 & 6? ;-)

i.e. "... this list of conditions, and the following disclaimer"

**Comment by philhodge** [comment]
Yes!

File: LICENSE.rst (hunk 1 above)
Comment (arfon) [inline LICENSE.rst]: Sorry to be a pedant but we shouldn't modify *at all* the standard license text. Even if we think the punctuation is...lacking.

Comment (justincely) [inline LICENSE.rst]: haha, @jamienoss supports both options - thanks!

**Comment by justincely** [comment]
official word from BRC is AURA for the copyright - so we should use that on all projects

**Pull request merged**